src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.TagContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tag contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FilterCommand extends Command {
+
+    public static final String COMMAND_WORD = "filter";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
```

Comment:
I believe that the filter command should be for tags, not names of people?

Reply:
Oops good eye I copied and pasted some parts from `FindCommand.java` and must have forgotten to change this part. Thank you!